packages/sonata-extra-bundle/Resources/public/main.js

```diff
@@ -0,0 +1,9 @@
+
+function setTimezoneCookie(environment) {
```

Comment:
Chance this to load automatically if the file is include.
Check if we are on https and don't use the path /admin.

rename the cookie for adminUserTimezone